### DISCUSSION TITLE
Welcome to PBI-InspectorV2 Discussions!

### DISCUSSION THREAD

**Discussion by NatVanG** [NatVanG/fab-inspector#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jjarrinf** [comment]
Hello, I am a Software Engineer working on a Data Product Build Project, and trying to get some common software development practices to our Data Analytics Developers using Power BI.

Power BI is not my development tool, actually I am far from being an expert, and I think this kind of tools require more diffusion.

I am integrating this tool in our pipelines, but I still cannot complete the "report" build due to this error:

##[error]PBI entry "reportLayout" with path "Report/Layout" is not valid or does not exist, resuming to next PBI Entry iteration if any

I tried with the V1 and the V2 set of report rules.. and still cannot complete the process.. so any advice will be welcomed..

Something I could help with is the translation to spanish of the rules template, someone did it with the datasets template so I think a similar job with the set of rules for reports were usable.

**Reply by NatVanG** [reply]
Hello, thanks for posting. If your Power BI developers do not save report with the enhanced metadata format (PBIR) which is currently in Preview then please use this version of PBI Inspector: https://github.com/NatVanG/PBI-Inspector. Please let me know if this helps.

**Reply by jjarrinf** [reply]
Hi, we are using the V2 and it works pretty fine now.. seems I did some configuration mistakes on my first try.  Thanks a lot.

**Comment by simonduz** [comment]
Hello and thank you for your contributions.   I have added PBI-InspectorV2 to my devops pipeline for deploying power bi .pbix, rdl, and pbip to the service.    I should have noticed before doing this that .pbix is not currently supported in V2 and I am wondering if you had a timeline to add .pbix and or .rdl files to this version.
I see version 1 supports .pbix and I am trying not to bloat my pipeline with version 1 for .pbix and version 2 for .pbip but if this is going to be the only option let me know.
Thanks again for this amazing tool.
